doc/architecture/sensor_view_configuration.adoc

```diff
@@ -1,9 +1,11 @@
 = Sensor view configuration
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu. Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a. Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+The sensor view configuration configures the sensor view.
+It is provided by the environment simulation.
```

Comment:
Maybe we should mention that it can be used in two different scopes:
1. It can be provided by the environment simulation, and explain.....
2. It can be provided by the sensor model, and explain....